plugins/repository-azure/build.gradle

```diff
@@ -54,7 +54,7 @@ dependencies {
   api "io.netty:netty-resolver-dns:${versions.netty}"
   api "io.netty:netty-transport-native-unix-common:${versions.netty}"
   implementation project(':modules:transport-netty4')
-  api 'com.azure:azure-storage-blob:12.20.0'
+  api 'com.azure:azure-storage-blob:12.20.1'
```

Comment:
It seems to be not related to the change, right?

Reply:
> > @reta just need your review and then we can merge or make changes as necessary
> 
> I have only concerns regarding this pattern we use in several places
> 
> ```
>                                 /*
>                                  * Making async synchronous for now.
>                                  */
>                                 inProgressIndexNameFuture.get(100, TimeUnit.SECONDS);
> ```
> 
> Otherwise LGTM (if you prefer to merge like this, we could address that later by chaining futures instead), thanks!

There are more latches coming in the next few PR's so let's address that when there are more of them.

Reply:
Yeah I don't think that's related to this change at all